var/spack/repos/builtin/packages/antlr/package.py

```diff
@@ -50,8 +50,11 @@ class Antlr(AutotoolsPackage):
     def configure_args(self):
         spec = self.spec
 
+        E = 'enable'
+        D = 'disable'
         return [
-            '--{0}-cxx'.format('enable' if '+cxx' in spec else 'disable'),
-            '--{0}-java'.format('enable' if '+java' in spec else 'disable'),
-            '--{0}-python'.format('enable' if '+python' in spec else 'disable')
+            '--{0}-cxx'.format(E if '+cxx' in spec else D),
+            '--{0}-java'.format(E if '+java' in spec else D),
+            '--{0}-python'.format(E if '+python' in spec else D),
```

Comment:
Aside from the `--disable-csharp` change, I would leave the rest of this package the same. PEP 8 says not to use capital letters in variable names. Also, single character variables names aren't very descriptive.

Reply:
> I would leave the rest of this package the same.

Unfortunately I can't.  Adding a comma to the '+python' line caused it to fail flake8 by going 1 character over the 79-char limit.  The changes above are a solution to that problem.



> PEP 8 says not to use capital letters in variable names.
PEP8 also says:
> Constants are... written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL.

> Also, single character variables names aren't very descriptive.

The names are defined, then used immediately over the next 4 lines.   Should not be a problem.  And if they were longer, it would defeat the purpose.

If there's another solution to the original problem that you like better, please suggest it.

Reply:
A few things:

* Autotools packages have access to the `enable_or_disable` function which should do this automatically for you
* More generally, for flake, you can split after the `format`:

```
         return [
             '--{0}-cxx'.format(E if '+cxx' in spec else D),
             '--{0}-java'.format(E if '+java' in spec else D),
             '--{0}-python'.format(
                 E if '+python' in spec else D),
             '--disable-csharp'
```

Reply:
Move `--disable-csharp` to the beginning so python is still last?